src/components/productlist.main.jsx

```diff
@@ -65,10 +123,22 @@ class ProductListMain extends React.Component {
   }
 
   render() {
-    const { categoryModel } = this.state;
+    const {
+      categoryModel,
+      isCompare,
+      compareList,
+      compareLink,
+    } = this.state;
     if (categoryModel._element && categoryModel._element.length > 0) {
       return (
         <div className="product-list-container" data-region="categoryBrowseRegion">
+          <div className="compare-button">
```

Comment:
Right now this compare button is displayed on the Category page *and* Search results page. Needs to only show up on Category page